### PR TITLE
set postgres + redshift incremental strategy

### PR DESCRIPTION
**Pull request by fivetran-jamie**
Pull Request
**Are you a current Fivetran customer?** 

fivetran made PR

**What change(s) does this PR introduce?** 

- [X] adjusts incremental strategy for postgres in the event models
- [X] updates changelog 
- [X] updates dbt_project.yml 
- [X] adds note about incremental materialization to readme
- [X]  twreaks PR template
- [x] regen docs 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🔪 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-dbt-redshift~=1.0.0
-dbt-postgres~=1.0.0
-dbt-spark~=1.0.0
-dbt-spark[PyHive]~=1.0.0
+dbt-snowflake>=1.0.0
+dbt-bigquery>=1.0.0
+dbt-redshift>=1.0.0
+dbt-postgres>=1.0.0
+dbt-spark>=1.0.0
+dbt-spark[PyHive]>=1.0.0
```
Comment (fivetran-catfritz) [inline integration_tests/requirements.txt:6]: Same question as for shopify.

File: models/klaviyo__events.sql [hunk 2]
```diff
@@ -6,7 +6,7 @@
             "field": "occurred_on",
             "data_type": "date"
         } if target.type == 'bigquery' else none,
-        incremental_strategy = 'merge' if target.type != 'snowflake' else 'delete+insert',
+        incremental_strategy = 'merge' if target.type not in ('snowflake', 'postgres', 'redshift') else 'delete+insert',
```
Comment (fivetran-catfritz) [inline models/klaviyo__events.sql:9]: Should the other models not merge with snowflake, or is it just specific this one that snowflake can't do merge?

File: integration_tests/requirements.txt (hunk 1 above)
Comment (fivetran-jamie) [reply integration_tests/requirements.txt:6]: adding ,<2.0.0

File: models/klaviyo__events.sql (hunk 2 above)
Comment (fivetran-jamie) [reply models/klaviyo__events.sql:9]: if i remember correctly it was just this model (i have no idea why though)

**Review by fivetran-catfritz — APPROVED** [review]

**Pull request merged**